CRM/Core/Resources/Common.php

```diff
@@ -97,7 +97,7 @@ public static function createStyleBundle($name) {
     $config = CRM_Core_Config::singleton();
     if (!empty($config->customCSSURL)) {
       $customCSSURL = Civi::resources()->addCacheCode($config->customCSSURL);
-      $bundle->addStyleUrl($customCSSURL, 99);
+      $bundle->addStyleUrl($customCSSURL, ['weight' => 99, 'name' => 'civicrm:css/custom.css']);
```

Comment:
(Suggestion)
Do we need the `:css` thingy? How about just making it `civicrm:custom.css`.
Its not a huge deal, but `civicrm:css/custom.css` kind of makes it looks like a URL. When `civicrm:custom.css` looks more of an identifier.

Reply:
Ah yes ... what's in a name??

Resources added via `addStyleFile()` have a default name combining the extension name and the relative path so the other resources added by `createStyleBundle()` are `civicrm:css/civicrm.css` and `civicrm:css/crm-i.css`.  So following that pattern of names I went with `civicrm:css/custom.css`. However,
1. the file isn't provided by core so arguably isn't `civicrm` - but it is added by core.
2. the path is not necessarily `css` - but is quite likely to be `[civicrm.files]/css`
3. the file name is not necessarily `custom.css`- but is a common option

Providing the name is unique I don't think it matters much.  The purpose is to let people target that custom css file, whatever it's actual filename or path.  I did consider `civicrm:custom.css` but thought the omission of the `css/` part might cause more question since the name then does not follow the pattern of the other names in the bundle.

Happy to change the name if there's consensus on a better one.

And while you're all thinking about this, perhaps you could also comment on https://lab.civicrm.org/dev/core/-/issues/2460 which led to this.

Reply:
Thanks for the explanation. It makes sense, no need to change in that case.